crates/oci-distribution/src/lib.rs

```diff
@@ -108,23 +113,26 @@ impl Client {
         }
     }
 
-    /// Pull a manifest from the remote OCI Distribution service.
-    ///
-    /// If the connection has already gone through authentication, this will
-    /// use the bearer token. Otherwise, this will attempt an anonymous pull.
-    pub async fn pull_manifest(&self, image: &Reference) -> OciResult<OciManifest> {
-        let client = reqwest::Client::new();
-        let url = image.to_v2_manifest_url();
-        let request = client.get(&url);
-
+    fn auth_headers(&self) -> HeaderMap {
```

Comment:
nit: might be more readable to have private functions come after public ones, but I'm not sure.

Reply:
Fixed